src/documentation/runners/mapreduce.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: "Apache Hadoop MapReduce Runner"
+permalink: /documentation/runners/mapreduce/
+redirect_from: /learn/runners/mapreduce/
```

Comment:
Shouldn't need this, since there's no past URL.

Reply:
done